.gitignore

```diff
@@ -0,0 +1 @@
+vendor
```

Comment:
* +1 blank line, missed that.
* I've seen so many permutations of this, `vendor`, `/vendor,  `/vendor/` `/vendor/*` and so on. Happy to change it, but do you happen to know a reasoning behind adding the trailing slash? Just to specify that it's a directory?
* `composer.phar` doesn't exist in the repo?